### ISSUE TITLE
Add input-group-sm for input-group-addon case

### ISSUE THREAD

**Issue by tamtakoe** [twbs/bootstrap#24201]
Now I can use `input-group-lg` only
```html
<div class="input-group input-group-lg">
  <span class="input-group-addon">@</span>
  <input type="text" class="form-control">
</div>
```
Add `input-group-sm` providing, please

**Comment by Johann-S** [comment]
Which release of Bootstrap do you use ?

**Comment by tamtakoe** [comment]
Last Bootstrap version 4.0.0-beta

**Comment by mdo** [comment]
This was fixed by #23151 in Beta 2. See https://getbootstrap.com/docs/4.0/components/input-group/#sizing.
